### DISCUSSION TITLE
Have anyone succeeded in setting Saleor up for production on linux server?

### DISCUSSION THREAD

**Discussion by D1ll1ng** [saleor/saleor#7209] [Deployment] [answered]
I have had a lot of problems with setting up a Saleor for production.

Have anyone succeeded in setting Saleor up for production on linux server?
Saleor 2.11 or 3.0.

If yes, do you mind share some information about what OS it's running on, and what hosting provider you use.

**Comment by karolkielecki** [comment, marked as answer]
@D1ll1ng Hi there! Please check [the following thread](https://spectrum.chat/saleor/general/release-deploy-saleor-production-deployment-bash~35ea562c-3d3f-4369-8140-8c74c6e11887). One of our community members has created the repo regarding deployment on the Ubuntu server https://github.com/thewhiterabbit/Deploy_Saleor
